_test/test_gen_sqw_workflow/gen_sqw_accumulate_sqw_tests_common.m

```diff
@@ -707,7 +716,7 @@ function test_accumulate_sqw11456(obj,varargin)
             
             [ok,mess,w2_11456,w2_11456acc]=...
                 is_cut_equal(sqw_file_11456,sqw_file_accum,obj.proj,...
-                [-1.5,0.025,0],[-2.1,-1.9],[-0.5,0.5],[-Inf,Inf]);
+                [-1.5,0.025,0],[-2.1,-1.9],[-0.5,0.5],[2,35]);
```

Comment:
What are these magic numbers? Why?

Reply:
these are the numbers related to initial range of input data. The change comes from the fixture, I did with correct img_db_ranges. I've added a sentence, vaguely explaining the situation.